packages/picker/src/Picker.ts

```diff
@@ -365,7 +365,11 @@ export class PickerBase extends SizedMixin(Focusable) {
                 @click=${this.onClick}
                 @sp-overlay-closed=${this.onOverlayClosed}
             >
-                <sp-menu id="menu" role="${this.listRole}"></sp-menu>
+                <sp-menu
+                    id="menu"
+                    selects="inherit"
```

Comment:
This is a little weird, since it has nothing to inherit, but it allows the menu to be unmanaged right now (so that `PickerBase` can continue managing until we actually finish reconciling this change).

Reply:
So `selects="inhereit"` is less controlled than `selects='null|undefined'`?

Reply:
That's true right now, but I'm thinking I should change this so that the `inherits` behavior is the default and `none` is explicit (which would disallow selection). This would also get rid of the difference between `sp-menu` and `sp-menu-group`. Originally I was thinking you'd always use a `menu-group` as the "sub-menu" pattern, but `menu-group` is really just adding a header. And now I'm wondering is `sp-menu-group` should even be it's own component...

Reply:
I like `selects="none"`!

I _could_ be convinced that `<sp-menu><span slot="heading">Title of Group</span></sp-menu>` was the same as `<sp-menu-group><span slot="heading">Title of Group</span></sp-menu-group>` for sure. Or maybe it could just be an extension, instead of a second completely independent component...

Reply:
On the one hand, I don't really want to add to this PR, but on the other hand if I could get that `sp-menu` out of the Shadow DOM, it would simplify the problem space quite a bit.

I'll open a separate issue for that so we can discuss.

Reply:
#1505